ibm_catalog.json

```diff
@@ -1094,33 +1076,23 @@
           ],
           "iam_permissions": [
             {
-              "role_crns": [
-                "crn:v1:bluemix:public:iam::::serviceRole:Manager"
-              ],
+              "role_crns": ["crn:v1:bluemix:public:iam::::serviceRole:Manager"],
```

Comment:
These lines shouldn't change... Not sure why all the iam_permissions Lines are being changed.. Please check